resources/views/components/layout/checkout.blade.php

```diff
@@ -0,0 +1,25 @@
+@slots(['header', 'sidebar'])
+<div class="fixed -top-px h-1.5 left-0 right-0 w-full bg-primary z-20"></div>
+<div {{ $attributes->class('text-ct-primary flex flex-wrap gap-x-20 text-sm max-md:flex-col') }}>
+    <div class="flex flex-wrap lg:flex-row flex-col flex-1 max-lg:pt-6">
```

Comment:
default flex behaviour is `flex-row` so you can change this to `max-lg:flex-col`